server/service/odp/listAndValidateOriginalDataPoints.ts

```diff
@@ -1,3 +1,13 @@
-import { OdpRepository } from '@server/repository/odp'
+import { OdpRepository } from '@server/repository'
+import { validateDataPoint } from '@common/validateOriginalDataPoint'
+import { CountryIso } from '@core/country'
+import { BaseProtocol, DB } from '@server/db'
 
-export const { listAndValidateOriginalDataPoints } = OdpRepository
+export const listAndValidateOriginalDataPoints = async (
+  options: { countryIso: CountryIso },
+  client: BaseProtocol = DB
+) => {
+  const { countryIso } = options
+  const odps = await OdpRepository.listOriginalDataPoints({ countryIso }, client)
+  return odps.map((odp: any) => ({ ...odp, validationStatus: validateDataPoint(odp) }))
```

Comment:
no any please,
also return types from all functions

Reply:
Return types will be added in next PR